src/sections/Projects/index.js

```diff
@@ -25,7 +25,7 @@ const Projects = () => {
                                 <div className="project__block__inner">
                                     <img src={projectImage1} alt="appion project" />
                                     <h3>Landscape</h3>
-                                    <p>Service Mesh Ecosystem</p>
+                                    <p>Service Mesh <br />Ecosystem</p>
```

Comment:
I dont think this will be a good move, as for views(mobile) where it can be fitted in a single line, it would cause an extra line break.